<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionFeeds</class>
 <widget class="QWidget" name="OptionFeeds">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OptionFeeds</string>
  </property>
  <widget class="QTableView" name="tableViewOptions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>851</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdFindSymbol">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Find Symbol</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSymbolName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listConsoleMsg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>851</width>
     <height>351</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Console Messages:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
